docs/core/additional-tools/dotnet-coverage.md

```diff
@@ -45,11 +45,12 @@ The `dotnet-coverage` tool:
 |-----------------------------------------------------------|
 | [dotnet-coverage collect](#dotnet-coverage-collect)       |
 | [dotnet-coverage merge](#dotnet-coverage-merge)           |
+| [dotnet-coverage snapshot](#dotnet-coverage-snapshot)     |
 | [dotnet-coverage shutdown](#dotnet-coverage-shutdown)     |
 
 ## dotnet-coverage collect
 
-The `collect` command is used to collect code coverage data for any .NET process and its subprocesses. For example, you can collect code coverage data for a console application or a Blazor application. This command is available on Windows (x86 and x64) and Linux (x64). The command supports only .NET modules. Native modules are not supported.
+The `collect` command is used to collect code coverage data for any .NET process and its subprocesses. For example, you can collect code coverage data for a console application or a Blazor application. This command is available on Windows (x86 and x64), Linux (x64), and macOS (x64). The command supports only .NET modules. Native modules are not supported.
```

Comment:
remove , before and

Reply:
@jakubch1 I think the comma should remain. But @gewarren can confirm.

Reply:
The Oxford comma is the approved style. This should stay.